spec/xcpretty/parser_spec.rb

```diff
@@ -225,6 +225,17 @@ module XCPretty
       @parser.parse(SAMPLE_TOUCH)
     end
 
+    it "parses write file" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.